### DISCUSSION TITLE
LineChartBarData loading data crash

### DISCUSSION THREAD

**Discussion by zhangyakai** [imaNNeo/fl_chart#1536] [Q&A]
Crash occurs when the following data is first loaded:
List<FlSpot>? datas;
    List<FlSpot> spots = [];
    for (int i = 0; i < datas!.length; i++) {
        spots.add(datas![i]);
    }

 LineChartBarData(
      spots: datas!,
      shadow: const Shadow(color: CMColor.blueColor),
      isCurved: true,
      gradient: getGradient2(CMColor.blueColor, const Color(0xFF5076AD)),
      barWidth: 3,
      isStrokeCapRound: true,
      dotData: const FlDotData(
        show: false,
      ),
      belowBarData: BarAreaData(
          show: true,
          gradient: getGradient2(CMColor.blueColor.withOpacity(0.3),
              const Color(0xFF5076AD).withOpacity(0.3))
      ),
    );

The data was passed in by me using the provider's data.
Crash exception:
* thread #10, name = 'io.flutter.1.raster', stop reason = EXC_BAD_ACCESS (code=1, address=0x0)
    frame #0: 0x00000001089f48e0 Flutter`impeller::ContentsFilterInput::GetCoverage(impeller::Entity const&) const + 4
Flutter`impeller::ContentsFilterInput::GetCoverage:
->  0x1089f48e0 <+4>:  ldr    x9, [x0]
    0x1089f48e4 <+8>:  ldr    x2, [x9, #0x18]
    0x1089f48e8 <+12>: br     x2
Flutter`impeller::FilterContentsFilterInput::~FilterContentsFilterInput:
    0x1089f48ec <+0>:  stp    x20, x19, [sp, #-0x20]!
Target 0: (Runner) stopped.

I want to know why this is? How should I solve it?

**Comment by imaNNeo** [comment]
Please add reproducible code
